### PR TITLE
Kratos mdpa tutorial for new version

### PR DESCRIPTION
**Pull request by e-dub**

### CODE REVIEW

File: tutorials/tutorial-6-k.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import pygem as pg
+from pygem.khandler import KHandler
+import vedo
+import meshio
+
+ffd = pg.FFD()
+ffd.read_parameters('../tests/test_datasets/parameters_test_ffd_pipe_unv_C0.prm')
+
+%cat ../tests/test_datasets/parameters_test_ffd_pipe_unv_C0.prm
```
Comment (mtezzele) [inline tutorials/tutorial-6-k.py:9]: are you sure this line works in a python script?

File: tutorials/tutorial-6-k.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+handler = KHandler()
+mesh_points = handler.parse('../tests/test_datasets/test_pipe.k')
+
+new_mesh_points = ffd(mesh_points)
+
+%cat ../tests/test_datasets/parameters_test_ffd_pipe_unv_C1.prm
```
Comment (mtezzele) [inline tutorials/tutorial-6-k.py:16]: same here

File: tutorials/tutorial-7-mdpa.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+new_mesh_points = ffd(mesh_points)
+
+DeformedMDPA = meshio.read('test_pipe_mod_C0.mdpa')
+meshio.write('test_pipe_deformed_mdpa.vtk', DeformedMDPA)
+DeformedVTK = vedo.load('test_pipe_deformed_mdpa.vtk')
+vedo.settings.embedWindow(backend='k3d', verbose=True)
+vedo.show(DeformedVTK, viewup="z", resetcam=True)
+
+%cat ../tests/test_datasets/parameters_test_ffd_pipe_unv_C1.prm
```
Comment (mtezzele) [inline tutorials/tutorial-7-mdpa.py:28]: same here

File: tutorials/tutorial-6-k.py (hunk 1 above)
Comment (e-dub) [reply tutorials/tutorial-6-k.py]: Works from Sypder, but not from the command line.  Therefore now gone! Thanks

File: tutorials/tutorial-6-k.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import pygem as pg
+from pygem.khandler import KHandler
+import vedo
```
Comment (mtezzele) [inline tutorials/tutorial-6-k.py:3]: Please keep only the used packages

File: tutorials/tutorial-6-k.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import meshio
```
Comment (mtezzele) [inline tutorials/tutorial-6-k.py:4]: Same here

File: tutorials/tutorial-6-k.py (hunk 2 above)
Comment (e-dub) [reply tutorials/tutorial-6-k.py]: agreed, removed from k file

File: tutorials/tutorial-7-mdpa.py (hunk 3 above)
Comment (e-dub) [reply tutorials/tutorial-7-mdpa.py]: I think it is good here as the mdpa can therefore be visualized

Comment (mtezzele) [reply tutorials/tutorial-7-mdpa.py]: Yes, for the external packages. I was referring to the `cat` line. Thanks

**Comment by mtezzele** [comment]
Dear @e-dub just a curiosity: you are able to visualize something in the file tutorial-7-mdpa.ipynb? From github I cannot see any plot...

**Comment by e-dub** [comment]
> Dear @e-dub just a curiosity: you are able to visualize something in the file tutorial-7-mdpa.ipynb? From github I cannot see any plot...

Chiaro! 
Screenshot_20201007_115447

**Comment by mtezzele** [comment]
@e-dub thanks for this contribution!

**Pull request merged**